tests/bundle/Functional/SearchView/SearchViewTest.php

```diff
@@ -74,6 +74,40 @@ public function testSimpleXmlContentQuery(string $xmlQueryBody, int $expectedCou
         self::assertEquals($expectedCount, $this->getQueryResultsCount('xml', $body));
     }
 
+    public function testLocationsByIdQueryContainsCurrentVersionObject(): void
+    {
+        $body = <<< JSON
+{
+    "ViewInput": {
+        "identifier": "locations-by-id",
+        "public": "false",
+        "LocationQuery": {
+            "Filter": {
+                "LocationIdCriterion": "2"
+            },
+            "limit": "10",
+            "offset": "0"
+        }
+    }
+}
+JSON;
```

Comment:
Since this is PHP 7.3+, you can indent HEREDOC / NOWDOC.

See https://www.php.net/manual/en/migration73.new-features.php#migration73.new-features.core.heredoc

> More Flexible Heredoc and Nowdoc Syntax ¶
> The closing marker for doc strings is no longer required to be followed by a semicolon or newline. Additionally the closing marker may be indented, in which case the indentation will be stripped from all lines in the doc string.